### ISSUE TITLE
Import path for linear transition not converted for ESM

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#185]
The "Add animation" commit c57d883be3784f2ab535b653aa0cbe30ae799302 breaks https://github.com/jimafisk/plenti_multi-pager_example test site.

<details>
<summary>Browser errors</summary>
<br>

Chrome:
```
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```
Firefox:
```
Loading module from “http://localhost:3000/spa/web_modules/svelte/internal/” was blocked because of a disallowed MIME type (“text/html”).
```

</details>

<details>
<summary>Screenshot</summary>

![linear](https://user-images.githubusercontent.com/5913244/161621459-80f65b0f-24c3-420e-8fff-2dbe5d84d253.png)

</details>

Gopack does not seem to be correctly converting the following import from http://localhost:3000/spa/web_modules/svelte/easing/index.mjs:

```js
export { identity as linear } from '../internal';
```

**Comment by jimafisk** [comment]
Current regex seems to be picking up the import:

<details>

```go
package main

import (
    "fmt"
  	"regexp"
)

func main() {
  string1 := "export { identity as linear } from '../internal';"
  re := regexp.MustCompile(`export(\s)(.*from(.*);|((.*\n){0,}?)\}(\s)from(.*);)`)
  fmt.Println(re.FindAllString(string1, -1))
}
```

Prints `[export { identity as linear } from '../internal';]` (see https://www.cattle.sh/#/p/1dgmedwko)
</details>

**Comment by jimafisk** [comment]
`public/spa/web_modules/svelte/transition/index.mjs` calls `runPack()` to handle the easing import:
```js
import { cubicInOut, linear, cubicOut } from '../easing';
```
...but `fullPathStr` (to be used as `convertPath` in recursive call) is set to `"public/spa/web_modules/svelte/easing"` which is missing the `.js` file extension.

**Comment by jimafisk** [comment]
We check the `fullPathStr` when building the `alreadyConvertedFiles` array and deciding if we want to recursively call `runPack()` on the import path. For relative paths that were pointing to a folder (or omitting a file ext) used for NPM, we needed to update the `fullPathStr` so we can rerun GoPack on the entrypoint of the imported file (not just a vague path to its folder).

**Comment by jimafisk** [comment]
Fix in [v0.5.3](https://github.com/plentico/plenti/releases/tag/v0.5.3)
